hack/create-custom-cloud-provider-config.sh

```diff
@@ -26,11 +26,12 @@ source "${REPO_ROOT}/hack/common-vars.sh"
 make --directory="${REPO_ROOT}" "${KUBECTL##*/}"
 
 # Test cloud provider config with shorter cache ttl
-CLOUD_PROVIDER_CONFIG="https://raw.githubusercontent.com/kubernetes-sigs/cloud-provider-azure/master/tests/k8s-azure/manifest/cluster-api/cloud-config-vmss-short-cache-ttl.json"
+CLOUD_PROVIDER_CONFIG="file://$(realpath ${REPO_ROOT})/templates/test/ci/default-custom-cloud-provider-config.json"
```

Comment:
`realpath` normalizes `$REPO_ROOT` into a plain absolute path instead of something relative like `hack/../hack/..`